locales/en/recommendations.ftl

```diff
@@ -0,0 +1,140 @@
+
+# Recommendation subhead
+rec-ssn-subhead = Review your credit reports
+# Link title
+rec-ssn-cta = Request credit reports
+rec-ssn = 
+  You get three free credit reports a year by law. 
+  Requesting and reviewing them does not affect your credit. 
+  Look for accounts, loans, or credit cards you don’t recognize.
+
+# Recommendation subhead
+rec-pw-1-subhead = Change your password
+# Link title
+rec-pw-1-cta = Change password for this site
+rec-pw-1 = 
+  Make this password unique and different from any others you use. 
+  A good strategy to follow is to combine two or more unrelated 
+  words to create an entire passphrase.
+
+# Recommendation subhead
+rec-pw-2-subhead = Update other logins using the same password
+# Link title
+rec-pw-2-cta-fx = View logins in { -brand-name }
```

Comment:
We can use a [`WebChannel`](https://developer.mozilla.org/en-US/docs/Mozilla/JavaScript_code_modules/WebChannel.jsm) to have this open `about:logins` though it needs a corresponding patch in Firefox.